modules/payloads/singles/cmd/unix/adduser.rb

```diff
@@ -0,0 +1,76 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+module MetasploitModule
+  CachedSize = 8
```

Comment:
Why this line?

Reply:
CachedSize? No idea. It was on the payload that I used as a template

Reply:
The `CachedSize` is used by the framework to determine if a payload is compatible with an exploit modules size limitations without having to generate it. The value is validated by CI, so the tests will fail if the generated size doesn't match this value.

Reply:
@smcintyre-r7 Where can I find more documentation on that? Do I need to use a function for it? Do I just remove it?

Reply:
I believe CI will reject this manual change; the tool to calculate the payload size needs to be updated. If that's the case, in the interim this value should remain as 8